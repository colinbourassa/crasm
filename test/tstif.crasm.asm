;;; Author: Leon Bottou
;;; Public Domain.

	* equ $2000

	dw end1-4*2
end2	dw (end1-4)*2
end1	dw $ad
	
	if 1<2
	  if 1>2
	    dl 3000
	  else
	    dw 4000
	  endc
	  dw 1000
	else
	  if 1<>1
	    dw 6000/2
	  endc
	  dw 2000
	endc

	clist off
	
	if 1>2
	  dw 3000
	else
	  dw 4000
	endc
	
	if 1<=1
	  dw 5000
	  if 1>2
  	    dw 3000
	  else
	    dw 4000
	  endc
	endc
	
	
