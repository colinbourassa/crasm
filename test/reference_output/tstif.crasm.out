S11320001FFC400000AD0FA003E80FA013880FA031
S9030000FC
